### ISSUE TITLE
DAC 14 - Medium Priority WCAG Level AA - Colour Contrast [Non-Text] 01

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#14]
<img width="445" alt="DAC_Colour_Contrast_Non_Text_01" src="https://user-images.githubusercontent.com/75850106/155557817-c1f302a7-12b0-4196-80e0-e2dcf3af4882.png">

### Issue
Low vision users may be unable to see the grey interactive dots within the interactive element beneath the heading ‘Income deprivation in Adur’. This is because the contrast of the elements against adjacent colours is less than the 3:1 minimum. This applies to any other diagrams on the page that have the same colour usage.
Foreground: #D3D3D3 Background: #FFFFFF
The contrast ratio is: 1.5:1 1.4.11 Non-text Contrast (AA)
Fail for UI components and graphical objects

<img width="606" alt="DAC 14 Current code ref" src="https://user-images.githubusercontent.com/75850106/155558011-e9176929-1387-4c43-b44f-95b99ff281d7.png">

**Low vision comments:**
“The grey dots on the location chart fail non-text colour contrast at an approximate ratio of 1.5:1, ideally this should be a ratio of 3:1 at least.”

### Suggested Solution
Ensure that non-text colour contrasts are at a minimum ratio of 3:1 against adjacent colours.
